packages/api/src/useFetch.ts

```diff
@@ -0,0 +1,12 @@
+import { useQuery } from 'react-query';
+import { getQueryKeys, send } from './utils';
+import type { TSocketEndpointNames, TSocketAcceptableProps, TSocketResponseData } from '../types';
+
+const useFetch = <T extends TSocketEndpointNames>(name: T, ...props: TSocketAcceptableProps<T, true>) =>
```

Comment:
are we caching in useFetch? or is it yet to be implemented :D

Reply:
@balakrishna-deriv Yes it's cashed, But still sending the request and updating the data in the background, We can also pass an options argument to the `useFetch` hook to control the behavior.

https://user-images.githubusercontent.com/72082844/227099083-acbb213d-efab-4fbe-b373-2e7fdfb239ac.mov